ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -255,14 +292,18 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
 
   validateEmailAddressPresence() {
     const {institution} = this.state;
-    return this.state.institution.duaTypeEnum === DuaType.MASTER ||
-      institution.emailAddresses && institution.emailAddresses.length > 0;
+    const rtEmailAddresses = getRegisteredTierEmailAddresses(institution);
+    return institution.tierRequirements && institution.tierEmailAddresses &&
```

Comment:
This is getting a little confusing so it would help to group these conditions for clarity.  Maybe something like:

```
const noAddressesRequired = institution.tierRequirements && getRegisteredTierRequirement(institution).membershipRequirement !== InstitutionMembershipRequirement.ADDRESSES;
const hasAddresses = institution.tierRequirements && getRegisteredTierRequirement(institution).membershipRequirement === InstitutionMembershipRequirement.ADDRESSES && rtEmailAddresses && rtEmailAddresses.length > 0;
return noAddressesRequired || hasAddresses
```

or

```
if (!institution.tierRequirements) {
  return false;
}
if (getRegisteredTierRequirement(institution).membershipRequirement === InstitutionMembershipRequirement.ADDRESSES) {
  const rtEmailAddresses = getRegisteredTierEmailAddresses(institution);
  return rtEmailAddresses && rtEmailAddresses.length > 0;
} else {
  return true;
}
```

Same comment for domains.